debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/stackwalker/MethodMetaData.java

```diff
@@ -516,6 +516,27 @@ public void jitAddSpilledRegisters(WalkState walkState, VoidPointer stackMap) th
 					}
 					while (! savedGPRs.eq(0));
 				}
+			} else if (J9ConfigFlags.arch_aarch64) {
+				savedGPRs = registerSaveDescription.bitAnd(new UDATA(0xFFFF0000));
```

Comment:
What is the intent here? If the goal is to select just 16 bits, the constant must change to `0xFFFF0000L`.
```
savedGPRs = registerSaveDescription.bitAnd(0xFFFF0000L);
```
If we want 24 bits, the mask should be `0xFFFFFFFFFFFF0000L`.
```
savedGPRs = registerSaveDescription.bitAnd(0xFFFFFFFFFFFF0000L);
```
Also notice there's no need to construct the intermediate `UDATA` object.

Reply:
I changed the constant to `0xFFFF0000L` as we need 16bit.